### DISCUSSION TITLE
Single point to multi point network

### DISCUSSION THREAD

**Discussion by FLYBYME** [svpcom/wfb-ng#95] [Questions]
Is it possible to have a single transmitter transmit independent streams to two different receivers?

**Comment by svpcom** [comment]
Yes. You should setup separate radio ports ( "-p" option for wfb_tx and wfb_rx) for each stream.
Tx will have two wfb_tx processes: one for first stream and one for second.

**Comment by FLYBYME** [comment]
So I dont have to have different shared keys.

Im looking to have a network something like:
transmitter<-->UAV<-->UAV

**Comment by svpcom** [comment]
Yes, but I recommend to use different key pairs for different drones for security
